app/views/votes/_voting.html.erb

```diff
@@ -0,0 +1,14 @@
+<% if Vote.exists?(quote: quote, user: current_user) %>
```

Comment:
This is going to be an N+1 query. For every vote on the page we query to see if the user has voted for this quote or not. I'd like to see this be done better.